Samples/StoragePickers/cs-sample/MainWindow.xaml.cs

```diff
@@ -0,0 +1,288 @@
+// and more about our project templates, see: http://aka.ms/winui-project-info.
+
+namespace FilePickersAppSinglePackaged
+{
+
+    [JsonSerializable(typeof(Dictionary<string, List<string>>))]
+    internal partial class SourceGenerationContext : JsonSerializerContext
+    {
+    }
+
+    public sealed partial class MainWindow : Window
+    {
+        public MainWindow()
+        {
+            this.InitializeComponent();
+            Title = "File Pickers Sample App";
+            
+            // Set window size to make it taller
+            this.AppWindow.Resize(new Windows.Graphics.SizeInt32(1800, 1100));
+        }
+
+        #region Helper Methods
+
+        private void LogResult(string message)
+        {
+            ResultsTextBlock.Text = $"[{DateTime.Now:HH:mm:ss}] {message}\n{ResultsTextBlock.Text}";
+        }
+
+        private Microsoft.Windows.Storage.Pickers.PickerLocationId GetSelectedNewLocationId()
+        {
+            switch (StartLocationComboBox.SelectedIndex)
+            {
+                case 0: return Microsoft.Windows.Storage.Pickers.PickerLocationId.DocumentsLibrary;
+                case 1: return Microsoft.Windows.Storage.Pickers.PickerLocationId.ComputerFolder;
+                case 2: return Microsoft.Windows.Storage.Pickers.PickerLocationId.Desktop;
+                case 3: return Microsoft.Windows.Storage.Pickers.PickerLocationId.Downloads;
+                case 4: return (Microsoft.Windows.Storage.Pickers.PickerLocationId)4;
+                case 5: return Microsoft.Windows.Storage.Pickers.PickerLocationId.MusicLibrary;
+                case 6: return Microsoft.Windows.Storage.Pickers.PickerLocationId.PicturesLibrary;
+                case 7: return Microsoft.Windows.Storage.Pickers.PickerLocationId.VideosLibrary;
+                case 8: return Microsoft.Windows.Storage.Pickers.PickerLocationId.Objects3D;
+                case 9: return Microsoft.Windows.Storage.Pickers.PickerLocationId.Unspecified;
+                case 10: return (Microsoft.Windows.Storage.Pickers.PickerLocationId)10;
+                default: throw new InvalidOperationException("Invalid location selected");
+            }
+        }
+
+        private Microsoft.Windows.Storage.Pickers.PickerViewMode GetSelectedNewViewMode()
+        {
+            switch (ViewModeComboBox.SelectedIndex)
+            {
+                case 0: return Microsoft.Windows.Storage.Pickers.PickerViewMode.List;
+                case 1: return Microsoft.Windows.Storage.Pickers.PickerViewMode.Thumbnail;
+                case 2: return (Microsoft.Windows.Storage.Pickers.PickerViewMode)2;
+                default: throw new InvalidOperationException("Invalid view mode selected");
+            }
+        }
+
+        private string[] GetFileFilters()
+        {
+            string input = FileTypeFilterInput.Text?.Trim() ?? "";
+            if (string.IsNullOrEmpty(input))
+                return new string[] { "*" };
```

Comment:
Is collection expression support in this csharp project? If so, do we consider using that as a best practice? e.g. here would be `["*"]`?